### PR TITLE
Deliverables meeting template

### LINKED ISSUE #148: Create a bi-weekly deliverables review meeting
# Summary

In #144 we discussed creating a **deliverables review meeting**. The goal of this meeting would be to do the following things:

- Go over our major deliverables on the development board
  - Have discussion / refinement around deliverables that need conversation and planning
  - Encourage team members to "pick up" deliverables for development
  - Provide assistance to others that are stuck on some deliverables
  - Discuss which deliverables we should prioritize next
- Go over our major support items
  - Discuss how support has gone for the last 2 weeks
  - Hand off the Support Steward role to the next person
 
# User Stories

- As a team member, this meeting should tell me what are the most important issues to work on next, and which I should focus on myself.
- As the next Support Steward, this meeting should tell me any context information needed for pre-existing support tickets, and where to focus my efforts next.

# Acceptance criteria

We've got this process written up in the Team Compass describing when/how this meeting works.

# Tasks to complete

- [ ] Agree on the general structure and cadence of this meeting
- [ ] Create a calendar event for it
- [ ] Create a meeting notes template to follow
- [ ] Have the first meeting
- [ ] Make edits as necessary
- [ ] Write it all up in the Team Compass

### PR DESCRIPTION
**Pull request by choldgraf**
This is a draft issue template that we can use to plan and structure our deliverables sync meetings.

A few highlights:

- Proposes a structure for the deliverables meeting with checklists for things to do.
- Proposes that we use **every other Monday** for this meeting at 8:30AM California time (which comes directly after our monthly team meeting...this is intentional, but let me know if this is too much).
- Reduces our monthly team meeting from 2 hours to 1.5 hours, with the idea that we'll use that last half hour for the deliverables review meeting.

closes #148

### CODE REVIEW

**Comment by sgibson91** [comment]
LGTM - just wanted to check on timings as this looks like it'd last an hour, but I thought we'd discussed 30mins?

**Comment by choldgraf** [comment]
Good point. Do folks think we can cover that in 30min? If so we could start with that and see how it goes

**Comment by sgibson91** [comment]
I think part of the reason to keep it short is so that we don't get too stuck into technical discussions. The point of these meetings is to sync up with what's done and prioritise what's next, not fix things there and then. But I think we can flex timings to find the rhythm of the team as we figure it out.

**Comment by choldgraf** [comment]
In that case I'll stick with 30 minutes since it'll be easier to grow the meeting as-needed rather than shrink it as-needed :-)

**Comment by choldgraf** [comment]
OK I've addressed comments, reduced this meeting to 30 minutes and added it to the end of our monthly team meetings. Updated the top comment with this new information. What do folks think?

**Comment by choldgraf** [comment]
How does the @2i2c-org/tech-team feel about having our first deliverables meeting on this Monday? Using the structure described here? We can prototype the meeting see how it goes or what we'd like to change?

**Comment by sgibson91** [comment]
I am definitely pro avoiding giant meetings where possible!

File: .github/ISSUE_TEMPLATE/meeting-deliverables.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+- **Calendar for future meetings:** [this Google Calendar](https://calendar.google.com/calendar/embed?src=c_4hjjouojd8psql9i1a8nd1uff4%40group.calendar.google.com&ctz=America%2FLos_Angeles)
+
+## Meeting agenda
+
+_Follow the agenda below, checking boxes as we complete each step._
+
+### Activity Backlog
+_(5m) Quick review of [Activity Backlog](https://github.com/orgs/2i2c-org/projects/5?fullscreen=true) to let team members signal-boost if needed._
+
+- [ ] Quick team report on what they're up to right now, where they could use help.
```
Comment (damianavila) [inline .github/ISSUE_TEMPLATE/meeting-deliverables.md:24]: I think there is an overlapping between this point and the team sync issue... Maybe the meeting should happen every other **Tuesday** so everyone has time to look at other's reports and look into the board by themselves?

**Comment by damianavila** [comment]
> How does the @2i2c-org/tech-team feel about having our first deliverables meeting on this Monday? Using the structure described here? We can prototype the meeting see how it goes or what we'd like to change?

See my above comment about Tuesdays instead...

> Something else we could discuss is whether we'd like this meeting to be off-cycle with our monthly meetings so that we can meet a bit earlier, and thus be a bit more timezone friendly for all (this would also help us avoid a giant meeting on days with team meetings)

I do think having this off-cycled from the monthly meeting is a good idea...

**Comment by choldgraf** [comment]
The latest commit does a couple things:

- Changes the meeting time to Tuesday, off-cycle with our monthly team meetings.
- Updates the first item to reference the team sync since team members should have already filled this out
- I've also added a calendar entry to our team calendar for **next Tuesday at 8am Pacific** (here's [**a timezone converter**](https://arewemeetingyet.com/Los%20Angeles/2000-01-01/08:00/2i2c%20Team%20Meeting#eyJ1cmwiOiJodHRwczovL2hhY2ttZC5pby9ZNVNCTXhWN1I2Q01xemVUWGdtNWtBIn0=))

I think this is ready to go as long as others are OK with the timing etc. Once this is merged I'll create an issue for our team's first deliverables meeting!

**Review by sgibson91 — APPROVED** [review]
🚀

File: practices/meetings.md [hunk 2]
```diff
@@ -22,6 +26,14 @@ We use [this HackMD for our team meeting notes and agenda](https://hackmd.io/Y5S
 
 See the [Team Calendar page](team/calendar) for information about when and where the meetings are held.
 
+## Deliverables review meetings
+
+The 2i2c team meets **every other Monday** for 30 minutes.
```
Comment (damianavila) [inline practices/meetings.md:31]:
```suggestion
The 2i2c team meets **every other Tuesday** for 30 minutes.
```

**Review by damianavila — APPROVED** [review]
LGTM now!

**Comment by choldgraf** [comment]
ok! let's close this and I look forward to chatting with you all about deliverables tomorrow :-) (except for @damianavila who is hopefully hanging out in the middle of nowhere ;-) )

**Pull request merged**